index.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
```

Comment:
Also has said by @fatukunda we should set es6 import statements i.e `import express from express`
You can setup babel and webpack to do this or use an npm package called https://www.npmjs.com/package/esm. 
Otherwise great work here